test/unittest/tests/test_extent_list.cpp

```diff
@@ -0,0 +1,425 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#include <memory>
+
+#include <catch2/catch.hpp>
+
+#include "wt_internal.h"
+
+struct ExtentWrapper {
+    ExtentWrapper(WT_EXT *raw) : _raw(raw) {}
```

Comment:
Make the constructor `explicit` when there is only one argument. Same applies for the `SizeWrapper` class.